debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -696,13 +696,13 @@ public static AutoCreateMode parse(String value, String defaultValue) {
             .withWidth(Width.SHORT)
             .withImportance(Importance.MEDIUM)
             .withDescription("The criteria for running a snapshot upon startup of the connector. "
-                    + "Options include: "
-                    + "'always' to specify that the connector run a snapshot each time it starts up; "
-                    + "'initial' (the default) to specify the connector can run a snapshot only when no offsets are available for the logical server name; "
-                    + "'initial_only' same as 'initial' except the connector should stop after completing the snapshot and before it would normally start emitting changes;"
-                    + "'never' to specify the connector should never run a snapshot and that upon first startup the connector should read from the last position (LSN) recorded by the server; and"
-                    + "'exported' deprecated, use 'initial' instead; "
-                    + "'custom' to specify a custom class with 'snapshot.custom_class' which will be loaded and used to determine the snapshot, see docs for more details.");
+                    + "Select one of the following snapshot options: "
+                    + "'always': The connector runs a snapshot every time that it starts; "
+                    + "'initial' (default):  If the connector does not detect any offsets for the logical server name, it runs a snapshot that captures the current full state of the configured tables. After the snapshot completes, the connector begins to stream changes from the binlog. "
+                    + "'initial_only': The connector performs a snapshot as it does for the  'initial' option, but after completing the snapshot, it does not stream changes from the redo logs.; "
+                    + "'never': The connector does not run a snapshot. Upon first startup, the connector immediately begins reading from the beginning of the binlog. "
+                    + "'exported': deprecated, use 'initial' instead; "
+                    + "'custom': to specify a custom class with 'snapshot.custom_class' which will be loaded and used to determine the snapshot, see docs for more details.");
```

Comment:
```suggestion
                    + "'exported': This option is deprecated; use 'initial' instead.; "
                    + "'custom': The connector loads a custom class  to specify how the connector performs snapshots. For more information, see Custom snapshotter SPI  in the PostgreSQL connector documentation.");
```

If it's possible to add a link, you could hyperlink **Custom snapshotter SPI** to https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-custom-snapshot

Reply:
@Naros I agree on that one. But it is a low priority from my PoV.

Reply:
Added [DBZ-6378](https://issues.redhat.com/browse/DBZ-6378).